src/scripts/main.js

```diff
@@ -1,3 +1,50 @@
 'use strict';
 
-// write code here
+function convertToNumber(str) {
+  return str.split('').filter(element => {
+    if (isNaN(+element)) {
+      return false;
+    }
+
+    return true;
+  }).join('');
+}
+
+const listOfEmployees = document.querySelector('ul');
+
+function sortList(list) {
+  const children = list.children;
+  const employees = [...children];
+
+  employees.sort((employee1, employee2) => {
+    const salary1 = employee1.dataset.salary;
+    const salary2 = employee2.dataset.salary;
+
+    return convertToNumber(salary2) - convertToNumber(salary1);
+  });
+
+  list.innerHTML = `
```

Comment:
try use insertAdjacentHTML

Reply:
But I need to replace html rather than add it. Can't I use innerHTML in this case?

Reply:
You can leave default list